x-pack/plugins/ml/public/application/services/ml_api_service/trained_models.ts

```diff
@@ -81,10 +78,10 @@ export function trainedModelsApiProvider(httpService: HttpService) {
      * @param params - Optional query params
      */
     getTrainedModelStats(modelId?: string | string[], params?: InferenceStatsQueryParams) {
-      const model = (Array.isArray(modelId) ? modelId.join(',') : modelId) || '_all';
+      const model = Array.isArray(modelId) ? modelId.join(',') : modelId;
```

Comment:
I've changed this to no use `_all` when requesting all jobs to be consistent with all of our other client side endpoint functions.
using `_all` will work if calling kibana our endpoints, unless it is specified with other ids. in that case an error will be returned saying there are no jobs/models with the ID `_all`. This is inline with es behaviour.